src/main/java/be/seeseemelk/mockbukkit/attribute/Attributes.java

```diff
@@ -0,0 +1,36 @@
+package be.seeseemelk.mockbukkit.attribute;
+
+import com.google.common.collect.ImmutableMap;
+import org.bukkit.attribute.Attribute;
+
+import java.util.Map;
+
+public class Attributes
```

Comment:
```suggestion
public class AttributeMock
```
Might be a better name for the Class since we are Adding the Values for org.bukkit.attributes.Atrribute

Reply:
Yeah that sounds good